src/TestFramework/PhpSpec/Adapter/PhpSpecAdapter.php

```diff
@@ -41,6 +41,6 @@ public function testsPass(string $output): bool
 
     public function getName(): string
     {
-        return TestFrameworkTypes::PHPSPEC;
+        return 'PhpSpec';
```

Comment:
Actually, looks like different styles are used:

* http://www.phpspec.net/en/stable/manual/upgrading-to-phpspec-4.html
* http://www.phpspec.net/en/stable/manual/upgrading-to-phpspec-3.html